tasks/sep/006-0-sc-guardian-updates/input.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
I want to try and recreate this `input.json` file myself. 

What branch on the optimism repo did you use? 


This command doesn't work for me: 
```
SYSTEM_CONFIG_START_BLOCK=4071248 go run op-chain-ops/cmd/op-upgrade/main.go --l1-rpc-url https://eth-sepolia.g.alchemy.com/v2/5ief20x2sn8ULZYkzYm2UtVeE3M1en9y --chain-ids 11155420 --superchain-target sepolia --outfile input.json 
```

Error: 
```
panic: Chain ID 11155420 is not allowed. We panic if the chain ID does not correspond to the mainnet or sepolia versions of Metal, Mode, or Zora. This is because OP Sepolia is currently on FPAC, which corresponds to OptimismPortal v3.3.0. However, we do not want to upgrade other chains to that yet. A proper fix to op-upgrade to allow specifying the targets versions of contracts is a large change, and we might end up deprecating op-upgrade anyway. Therefore, we instead hardcode the chain IDs this script can be used for and panic if the chain ID is not one of them. This way it's not possible to erroneously upgrade a chain to an unexpected version.
```

Reply:
I made it partially with the [txn builder](https://app.safe.global/apps/open?safe=sep:0xa87675ebb9501C7baE8570a431c108C1577478Fa&appUrl=https://apps-portal.safe.global/tx-builder) and partly crafted by hand.

This runbook isn't making proxy changes, which is what `op-upgrade` is designed for. 
IMO validating the result of the input.json's execution is sufficient to validate its contents.